experiment_handling.py

```diff
@@ -95,17 +95,21 @@ def slave(slavecalcs, sid):
 
 
 #TODO: needs more efficient resave handling for every large runs / experiments
-def resave_data(save_path, parameter_combinations, index, eva, name,
-                sample_size=None, badmisskey=None):
+def resave_data(source_path, parameter_combinations, index, eva, name,
+                sample_size=None, badmisskey=None, save_path=None):
     """
     Resaves the computed "micro" data to smaller files that stores only "macro"
-    quanteties of interest. The pickle files are saved in the "data" directory
-    at this level of the filesystem tree.
+    quanteties of interest. If a save_path is given, the pickled procecced data
+    is saved separate from the raw input data.
 
     Parameters
     ----------
+    source_path : string
+        The path to the folder where the raw simulation data is stored
     save_path : string
```

Comment:
You could order the Parameters in the docs according to the ordering of the function definition above